tests/store/test_sqlalchemy_store.py

```diff
@@ -441,8 +450,10 @@ def test_log_null_metric(self):
         tval = None
         metric = entities.Metric(tkey, tval, int(time.time()))
 
-        with self.assertRaises(MlflowException) as exception_context:
+        warnings.simplefilter("ignore")
+        with self.assertRaises(MlflowException) as exception_context, warnings.catch_warnings():
```

Comment:
This silences a "null metric value" warning because we're deliberately testing that logging a null metric raises an exception